core/lib/Thelia/TaxEngine/TaxType/FeatureFixAmountTaxType.php

```diff
@@ -46,7 +47,9 @@ public function fixAmountRetriever(Product $product)
             ->findOne();
 
         if (null !== $query) {
-            $taxAmount = $query->getFreeTextValue();
+            if ( (1 == $taxAmount = $query->getFreeTextValue()) && !is_null($query->getFeatureAvId()) ) {
```

Comment:
Please use `getIsFreeText()` instead of  the depracted method `getFreeTextValue()`